stripe/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -521,5 +521,24 @@ data class ConfirmPaymentIntentParams internal constructor(
                 returnUrl = "stripe://return_url"
             )
         }
+
+
+        /**
+         * Create the parameters necessary for confirming a [PaymentIntent] with Blik
+         *
+         * @param blikCode code generated from blik mobile banking app
+         * @param clientSecret client secret from the PaymentIntent that is to be confirmed
+         *
+         */
+        @JvmStatic
+        fun createBlik(
+            blikCode: String,
```

Comment:
Where is `blikCode` used?

Reply:
it's generated by user's mobile banking app and merchant is supposed to collect the code through some UI and send it as a parameter when confirming the pi: https://youtu.be/0Lr8D5JoolE

Reply:
I don't see it being used in the method.

Reply:
oh right, the test uses default constructor to create one, removed this method